src/dirackernels/MyTRIMDiracSource.C

```diff
@@ -52,5 +54,5 @@ MyTRIMDiracSource::addPoints()
 Real
 MyTRIMDiracSource::computeQpResidual()
 {
-  return -_test[_i][_qp];
+  return -_test[_i][_qp] / _rasterizer.getTrimParameters().last_executed_dt;
```

Comment:
This does not seem right. Off the cuff I'd think it'd be ratio between last executed dt and current dt...

Reply:
`_test[_i][_qp] ` has units of defects but the kernel must have units of defects / time.

Reply:
Yeah, you're right